### PR TITLE
feat: Add a active user count on the top right on header

### LINKED ISSUE #18: feat:  Add a active user count on the top right on header
**Is your feature request related to a problem? Please describe.**

Currently visitor can't be able to see how many user are active now on the platform, so that they can match easily, as number of active user directly reflects the possibility of getting a match quickly.

**Describe the solution you'd like**

It is important to show the current active user, so that they can switch their comfort language accordingly to get a match quickly.

### PR DESCRIPTION
**Pull request by Er-luffy-D**
## Description
Added a new user count feature that retrieves the current number of users from the server's ' UserManager.js ' and displays it in a UserCount component. The component is integrated into the navbar, which will be accessible on the chat page.

Fixes #18 

## Screenshots:
Screenshot 2024-10-29 160026

## Changes

- Created a Navbar.tsx file in components folder
- Created a UserCount.tsx file to fetch the User count
- Added the Navbar component in the ChatPage.tsx
- Modified the index.js and UserManager.js

### CODE REVIEW

**Comment by Er-luffy-D** [comment]
Hey @kaali001  , could you kindly review this and let me know if there's anything wrong? I don't do much open-source work.

**Comment by Er-luffy-D** [comment]
Hey @kaali001 , please review this and let me know, is something wrong or all things are alright... Thanks 👍

File: server/index.js
Comment (kaali001) [inline server/index.js:1]: @Er-luffy-D ,  Remove all the changes you have made on this file ,  just keep your changed codes  at line 99 to 102.

File: client/src/pages/ChatPage.tsx
Comment (kaali001) [inline client/src/pages/ChatPage.tsx:1]: in this file , remove changes at line  95 -97, and 109.

>

File: client/src/Components/UserCount.tsx
Comment (kaali001) [inline client/src/Components/UserCount.tsx:1]: don't create a  new `Components` folder , just  use already present folder `components` to keep `Navbar.tsx` and `UsrCount.tsx`.  

Also fix this -  by first  time going to chat page it shows user count : Loading,  Actually it should show here `0` .

Comment (Er-luffy-D) [reply client/src/Components/UserCount.tsx:1]: Done

**Comment by kaali001** [comment]
Looks good to me, merging it now.

**Review by kaali001 — APPROVED** [review]

**Pull request merged**